model/ftn/w3initmd.ftn

```diff
@@ -954,6 +958,20 @@
         TOLAST(2,J) =        ODAT(J0+5)
       END DO
 !
+! CKECKPOINT J=8, for second stream of restart files
```

Comment:
While this is for checkpoint restarting for us, is this stream of restarts, really just a second stream or restarts instead of 'checkpoint' restarts only.  Theoretically we could have check point restarted before without this if the cadence of needed restarts were the same for what we needed for the next IC and the check point restarting

Reply:
@JessicaMeixner-NOAA this is a second stream of restart files that could be used for checkpointing. I will change the comments in the code to reflect this is the case.

Reply:
Changed and removed CKECKPOINT...

Reply:
@RobertoPadilla-NOAA added to thread for awareness